src/content/docs/faq/optional-mods.mdx

```diff
@@ -19,26 +19,30 @@ Sodium is a mod that improves the performance of Minecraft. It should be install
 Iris Shaders is a mod that adds shader support to our pack. It requires Sodium (and thus Indium) to be installed.
 It is not supported by Nvidium and some shader packs might not work with Distant Horizons.
 
-<Aside>Iris hasn't been added as it depends on Sodium.</Aside>
+<Aside>Iris hasn't been added by default as it depends on Sodium.</Aside>
 
 ## Distant Horizons
 
 Distant Horizons is a mod that improves the client side render distance by creating LODs beyond the real chunks' render distance.
 It only works with shaders built with Distant Horizons support in mind.
 
-<Aside>Distant Horizons is not enabled by default due to the LOD ComputerCraft monitors not rendering their display.</Aside>
+We have installed Distant Horizons on the server side in order to pre-render LODs for the client.
+
+<Aside>Distant Horizons is not enabled by default due to the LOD ComputerCraft monitors not rendering their display and because it is in alpha.</Aside>
```

Comment:
Swap the order of the Distant Horizons' asides, so the caution one introduces the notion of the mod being in alpha in a more smooth written way